### ISSUE TITLE
gitsync queries in tilt (?)

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#2210]
![image](https://user-images.githubusercontent.com/17987728/186896346-0382532f-540b-434f-a839-4fe279be66c2.png)

since yesterday night there has been no movement here; other types of queries get processed all right. (all gitsyncs went through yesterday and then stopped)

@duncdrum @line-o do you have an advice how to proceed best? 

i can kill the queries of course but then the deliveries won't make it to db - should i then upload and expand the files manually, redeliver through webhook (?) or are there other options?

(in some cases the changes did go through to BetMasData but not to expanded, in some cases they did not even get to BetMasData yet)

i am sorry, i know this is not your task, in case i will do it all manually, but maybe there is a trick i am not aware of

**Comment by duncdrum** [comment]
for now please do it manually, reviewing the git-sync task is very high on our ToDo list.

**Comment by eu-genia** [comment]
thank you, so I will kill the jobs and reload the data tonight

**Comment by eu-genia** [comment]
this issue https://github.com/BetaMasaheft/Documentation/issues/2218 made me realize what was wrong: the permissions for BetMasData and expanded were all off, users could not write there. Chmod seems to have fixed the thing. Was not the fault of the script.

**Comment by duncdrum** [comment]
@eu-genia yes the same process is also corrupting your backups. This is pretty high on the list of things I would like to address next. But first we need to finish the current round of refactoring, before we can locate the source of the problem in the codebase.

**Comment by eu-genia** [comment]
the permissions are all ok but since a while it takes ages for the gitsync to run (it used to be a matter of 1-2 minutes per file, now it is hours - it does finish by itself, but this is surely not normal, and this was not the case a month ago)
the upload to BetMasData itself works fast, but the following queries seem to be having difficulties they did not use to have. the permissions are in place.
i also noted that there have been no updates in the rdf repository on the app, makerdf seem to not run (the last uodate is from my manual run on 8/30)

**Comment by eu-genia** [comment]
the only edits i did to gitsync were on 8/17 and very minor (https://github.com/BetaMasaheft/BetMas/commit/8893a79a617a625483983741518aef30f850d080) ; the critical slowdown occured only several days ago so should not be related...

**Comment by eu-genia** [comment]
Noted now by comparing the versions in diff that all `console:log `functions have been commented out in `gitsync`, so the version in the app is not 100% the same as on github, but this was not me, am not sure why the `log `has been commented out and whether to update the github version

tested also commenting out `gitsync:rdf `since it does not seem to work anyway but this does not seem to speed it up

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/Documentation/issues/2766
https://github.com/BetaMasaheft/Documentation/issues/2528
